### DISCUSSION TITLE
Unable to see previewer in Visual Studiod Code

### DISCUSSION THREAD

**Discussion by tataDan** [AvaloniaUI/Avalonia#17518] [Q&A]
When I open `MainWindow.axaml` and then click on the `showPreview ` icon a window appears with the message "Previewer is not available. Build the project first.".  When I click on the `Build` button, nothing happens.

**Comment by tataDan** [comment]
It should be noted that I am running Debian 12 Linux.

**Comment by thevortexcloud** [comment]
No idea what is happening here, but Rider is now free to use (for non commercial use) and has much better integration with Avalonia. The previewer also works fine on Linux.

https://www.jetbrains.com/rider/buy/?section=personal&billing=yearly
